lib/schedule.js

```diff
@@ -327,7 +341,7 @@ Range.prototype.contains = function(val) {
 
   NOTE: Cron months are 1-based, but RecurrenceRule months are 0-based.
 */
-function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second) {
+function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second, timezone) {
   this.recurs = true;
```

Comment:
I think doing it that way makes things simpler, since you're just relying on CronDate for handling the addition of time steps until the next execution. 

Changing it to the other way might force you to make time-zone aware date elements when assigning them to the RecurrenceRule at scheduling.